src/Data/Arrays.php

```diff
@@ -363,10 +363,7 @@ public function subSetOfArray(array $array, $keys)
      */
     public static function forget(&$array, $keys, $opr = null)
     {
-        $arrOrg = $array;
-        if ($null !== $keys) {
-            return false;
-        }
```

Comment:
This should be useless.

When the `$keys` is null, it will be the empty `[]` array when casting to the array.